python/ray/serve/schema.py

```diff
@@ -186,6 +186,94 @@ def __eq__(self, other: Any) -> bool:
         return self._compute_hash() == other._compute_hash()
 
 
+@PublicAPI(stability="alpha")
+class TracingConfig(BaseModel):
+    """Tracing config for configuring serve tracing.
+
+    Example:
+
+    my_exporter.py:
+
+        .. code-block:: python
+
+        import os
+        from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+        from typing import List
+
+        def tracing_exporter() -> List[SimpleSpanProcessor]:
+            os.makedirs("/tmp/spans", exist_ok=True)
+            return [
+                SimpleSpanProcessor(
+                    ConsoleSpanExporter(out=open(f"/tmp/spans/{os.getpid()}.json", "a"))
+                )
+            ]
+
+    serve_config.yaml:
+
+        tracing_config:
+
+            exporter_import_path: my_exporter:tracing_exporter
+
+            enable: true
+
```

Comment:
why's the spacing weird here?

Reply:
This was the format when I built the serve_config. Updated